en_US/faq/concept.md

```diff
@@ -5,8 +5,8 @@
 EMQX has [3 products](https://www.emqx.com/en/products/emqx) in total. The different products support different number of connections, features, services, etc.
```

Comment:
We could also list Serverless and BYOC here as separate products. Or, rewrite this completely.
```suggestion
EMQX has [2 products](https://www.emqx.com/en/products/emqx) in total. The different products support different number of connections, features, services, etc.
```